c/CMakeLists.txt

```diff
@@ -50,7 +50,9 @@ cmake_minimum_required(VERSION 2.8)
 set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 
 # Some compiler options used globally
-set(CMAKE_C_FLAGS "-Wall -Werror -std=gnu99 ${CMAKE_C_FLAGS}")
+if (NOT MSVC)
+  set(CMAKE_C_FLAGS "-Wall -Werror -std=gnu99 ${CMAKE_C_FLAGS}")
```

Comment:
There is a target property for C standard, but unfortunately I can't find a portable one for warning level.  But yes, there are definitely MS equivalents for -Wall, etc.
https://docs.microsoft.com/en-us/cpp/build/reference/compiler-option-warning-level?view=vs-2017

Reply:
Updated. `std` not applicable, see https://docs.microsoft.com/en-us/cpp/build/reference/std-specify-language-standard-version?view=vs-2017